ql/src/test/queries/clientpositive/complex_type_to_string.q

```diff
@@ -0,0 +1,31 @@
+create table table1(id int, txt string, num int, flag string);
```

Comment:
Could you please add some test cases where complex types are nested?
```
struct {
  a map<string, array<int>>,
  b struct { c decimal(10,3), d double}
}
```

Reply:
Added nested test in udf_to_string.q